src/dto/flow-rata-run.dto.ts

```diff
@@ -167,6 +168,7 @@ export class FlowRataRunBaseDTO {
   )
   percentMoisture: number;
 
+  @IsOptional()
```

Comment:
All the properties can be nullable or optional.

![image](https://user-images.githubusercontent.com/102534033/231240553-c1b4100a-9966-4731-95b3-b3ad740644c4.png)